.gitignore

```diff
@@ -0,0 +1,3 @@
+#.gitignore
+test.md/
+node_modules/
```

Comment:
As BenMukebo pointed out, you did an excellent job ignoring the node_modules directory and the test.md file but the issue is that the test.md file is on your repo 😥 Kindly delete this file from your remote repository so that you meet the project requirements 💪